article/app/views/fragments/articleBody.scala.html

```diff
@@ -29,6 +29,7 @@
             "content--article",
             "tonal",
             s"tonal--${toneClass(article)}",
+            s"content--pillar-${article.metadata.pillar.nameOrDefault}",
```

Comment:
I this amp? I think this is for normal articles

Reply:
Ah it seems normal articles use articleBodyGarnett.scala.html and AMP is using articleBody...